App/Containers/OnboardingScreen/index.tsx

```diff
@@ -149,10 +149,6 @@ class OnboardingScreen extends React.Component<Props, State> {
         />
       )
     ]
-    // remove referral code if needed.
-    if (this.props.skipReferralCode) {
-      pages.splice(3, 1)
-    }
```

Comment:
No more dynamic splicing of the page data. It it always the same now.

Reply:
👍